itl/command-plugins.conf

```diff
@@ -423,8 +427,13 @@ object CheckCommand "http" {
 		}
 		"-C" = {
 			value = "$http_certificate$"
-			description = "Minimum number of days a certificate has to be valid. This parameter explicitely sets the port to 443 and ignores the URL if passed."
+			description = "Minimum number of days a certificate has to be valid. This parameter explicitely sets the port to 443 and ignores the URL if passed"
+		}
+		"--continue-after-certificate" = {
+			value = "$http_certificate_continue$"
```

Comment:
Since this option is a boolean flag, please use `set_if` instead of `value`. Icinga 2 will then only pass this argument if it is explicitly set to `true`.